model/pipeline/src/main/proto/org/apache/beam/model/pipeline/v1/external_transforms.proto

```diff
@@ -106,4 +109,13 @@ message BuilderMethod {
   bytes payload = 3;
 }
 
+message SchemaTransformPayload {
+  // Identifier of the SchemaTransform
+  string identifier = 1;
 
+  // The config of the SchemaTransform.
+  // Should be decodable via beam:coder:row:v1.
+  // The schema of the Row should be compatible with the schema of the
+  // SchemaTransform denoted by the identifier.
+  bytes config_row = 2;
```

Comment:
Should we also provide the schema that this is encoded against, in case the schema evolves (in a compatible way of course) between when the discovery happens and when this request is made.